calcom/cal.com
# fix: prevent creating multiple platform orgs

## What does this PR do?



- This PR is an enhancement to #16564 where in we redirect the user back to platform dashboard if they're trying to create another platform account. This adds a check to the organization creation handler which make sure the user can only create a single platform account, and if they have an already existing org or platform team created we don't end up creating another platform team.



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?
Go to `/settings/platform/new` in web app and setup a platform team; once you have a platform team created try to add another one by going into `/settings/platform/new`. You should see an error message saying `User is already part of a team`.
